vendor/github.com/aws/aws-sdk-go-v2/service/iam/api_client.go

```diff
@@ -0,0 +1,434 @@
+// Code generated by smithy-go-codegen DO NOT EDIT.
```

Comment:
Do the `smith-go-codegen` generated files have to be committed or can they be generated when building the SDK?

Reply:
I believe they have to be committed as the SDK vendors all dependencies